src/timeMachine/selectors/index.ts

```diff
@@ -293,14 +295,27 @@ export const getStartTime = (timeRange: TimeRange) => {
   }
 }
 
+// this function takes in a string that represents a date, and swaps the timezone to local.
+// if the date passed was 2021-09-02 12:00:00 UTC the function will return 2021-09-02 12:00:00 <Local TZ>
+const setTimeToLocal = (date?: string): Date => {
+  const localTime = date ? new Date(date) : new Date()
+  localTime.setMinutes(localTime.getMinutes() + getTimezoneOffset())
+
+  return localTime
+}
+
```

Comment:
I hate to do this so late in the game, but I think this method should be moved into `dateTimeUtils`

Reply:
Oh yes totally, i'll move it!